examples/android-material-ui/.storybook/preview.js

```diff
@@ -1,3 +1,34 @@
 import { DeviceDecorator } from "@storybook/native-addon";
+import {getGlobalLocationJson} from "@storybook/native-devices";
 
-export const decorators = [DeviceDecorator];
+
+const preview = {
+    globalTypes: {
+        // location: {
```

Comment:
Can you include a note indicating that this is just an example of using google maps and that's why its commented out?